packages/disputer/index.js

```diff
@@ -65,7 +65,7 @@ async function run({
 
     const collateralToken = new web3.eth.Contract(getAbi("ExpandedERC20"), collateralTokenAddress);
     const syntheticToken = new web3.eth.Contract(getAbi("ExpandedERC20"), syntheticTokenAddress);
-    const [currentAllowance, collateralTokenDecimals, syntheticTokenDecimals] = await Promise.all([
```

Comment:
changed this to be more consistent with how the variable is used downstream and between files.